Replace colons with a period

sunithasomanna opened the pull request:
Replacing colons with period for relevant parameters in the API readme.
Also removed extra spaces before the headings

sunithasomanna committed:
replacing colons with a period

sunithasomanna committed:
replacing colons with a period

sunithasomanna committed:
minor fixes with spacing and lining

sunithasomanna committed:
replacing more colons with a period

rohitkumar-ro reviewed (approved):
Changes looks fine

arahimanshaik reviewed (approved):
Looks good for me

Bharath-KKB commented on docs/README.md:
```diff
@@ -182,38 +182,29 @@
+  The plugins abstract, translate, and expose southbound resource information to the resource aggregator through RESTful APIs. HPE Resource Aggregator for ODIM supports:
+  
+  - Generic Redfish plugin for ODIM (GRF): Generic Redfish plugin that can be used as a plugin for any Redfish-compliant device.
+  - Plugin for unmanaged racks (URP): Plugin that acts as a resource manager for unmanaged racks. 
```
We are missing Cisco ACI, Lenovo and Dell plugins?

sunithasomanna replied:
Addressed all your comments on a different PR (https://github.com/sunithasomanna/ODIM/pull/21). Please review the updated document at https://github.com/sunithasomanna/ODIM/blob/sprint-47-odim-68/docs/README.md

Bharath-KKB commented on docs/README.md:
```diff
@@ -182,38 +182,29 @@
-are designed as per DMTF's [Redfish® Scalable Platforms API (Redfish) specification 1.11.1](https://www.dmtf.org/sites/default/files/standards/documents/DSP0266_1.11.1.pdf) and are fully Redfish-compliant.
-Redfish® is an open industry standard specification, API, and schema. It specifies a RESTful interface and uses JSON and OData. The Redfish
-standards are designed to deliver simple and secure environment for managing multivendor, converged, and hybrid IT infrastructure.
+This guide provides reference information for the northbound APIs exposed by the resource aggregator. These APIs are designed as per DMTF's [Redfish® Scalable Platforms API (Redfish) specification 1.11.1](https://www.dmtf.org/sites/default/files/standards/documents/DSP0266_1.11.1.pdf) and are fully Redfish-compliant.
```
Confirm if this is the right version

Bharath-KKB commented on docs/README.md:
```diff
@@ -227,48 +218,31 @@ The following figure shows these functional layers of Resource Aggregator for OD
 
 - **Services layer**
 
 The services layer is where all the services are hosted. This layer implements service logic for all use cases
```
This is wrong. Services and collections both provide functionality. Also the figure is that of collections... no services there

Bharath-KKB commented on docs/README.md:
```diff
@@ -227,48 +218,31 @@ The following figure shows these functional layers of Resource Aggregator for OD
 through an extensible domain model (Redfish Data Model). Requests coming from the API layer and the
-responses coming from the plugin layer are mapped to the actual end resources in this layer. It maintains the state
-for event subscriptions, credentials, and tasks. It also hosts a message bus called the Plug-in Message Bus (PMB).
+responses coming from the plugin layer are mapped to the actual end resources in this layer. It maintains the state for event subscriptions, credentials, and tasks. It also hosts a message bus called the Plug-in Message Bus (PMB).
```
What is a PMB?

Bharath-KKB replied:
Plus _" Requests coming from the API layer and the responses coming from the plugin layer are mapped to the actual end resources in this layer."_ is not correct.

Bharath-KKB replied:
This _"This layer hosts a message broker which acts as a communication channel between the upper layers and the plugin layer. It supports common messaging architecture and real-time streaming. Resource Aggregator for ODIM uses Kafka as the event message bus. The services and the event message bus layers host Redis data store."_ needs changes. This is only used for events. And arent we supporting redis streams ?

Bharath-KKB commented on docs/README.md:
```diff
@@ -227,48 +218,31 @@ The following figure shows these functional layers of Resource Aggregator for OD
-The plugin layer allows developers to create plugins on any tool set of their choice without enforcing any strict
-language binding. To know how to develop plugins, refer to [Resource Aggregator for Open Distributed
-Infrastructure Management Plugin Developer's Guide](https://github.com/ODIM-Project/ODIM/blob/development/plugin-redfish/README.md).
+This layer connects the actual managed resources to the aggregator layers and is de-coupled from the upper layers. A plugin abstracts vendor-specific access protocols to a common interface which the aggregator layers use to communicate with the resources. It uses REST-based communication which is based on OpenAPI Specification v3.0 to interact with the other layers. It collects events to be expos ...
```
We cannot claim OpenAPI v3 support. Also _"The messaging mechanism is based on OpenMessaging Specification."_ doesnt look right

Bharath-KKB reviewed (changes requested):
Pls fix the issues identified.

sunithasomanna commented:
Addressed all comments given by Bharath K, on a different PR (#21). Please review the updated document at https://github.com/sunithasomanna/ODIM/blob/sprint-47-odim-68/docs/README.md
